net/k6/Portfile

```diff
@@ -30,3 +32,5 @@ destroot {
     xinstall -m 0644 -W ${worksrcpath} README.md LICENSE.md \
         ${destroot}${docdir}
 }
+
+github.tarball_from archive
```

Comment:
What is this for?

Reply:
The default download URL format in the Github portgroup (`tarball`) is considered deprecated.  New Github these days generate the `archive` style URLs. Github is still supporting `tarball` style URLs, but it's in our best interest to shift to `archive`. So I've been updating ports to use `archive` progressively.